persistence-modules/pom.xml

```diff
@@ -43,7 +43,6 @@
         <module>java-jpa</module> <!-- long running -->
         <module>java-jpa-2</module> <!-- long running -->
         <module>java-jpa-3</module>
-        <module>java-mongodb</module> <!-- long running -->
```

Comment:
why remove from here?

Reply:
Because I included in the submodule fashion to run in with jdk 9 and above

Reply:
Let's comment it and add a comment <!-- enable it when persistence-modules is migrated to JDK9+ -->

Reply:
Done